eip-template.md

```diff
@@ -1,6 +1,7 @@
 ---
 eip: <to be assigned>
 title: <EIP title>
+description: <Provide a simplified and layman-accessible explanation of the EIP.>
```

Comment:
I think it would be valuable to include this text or similar, but not a hard requirement if others disagree:
> Imagine an email subject line, GitHub PR title, or forum post title.

Reply:
I don't want to argue too much about it, but what is the title then?

Reply:
Title is a few words, not a complete sentence.
> King Micah

Description is one full (short) sentence.
> The benevolent dictator of the EIPs repository.

Abstract is multi-sentence (short paragraph) technical summary.
> Micah is the unelected, self assigned dictator of the EIPs repository.  They rule by way of blocking an PRs they don't like and merging things they like without waiting for agreement from anyone else.  Micah's power can be taken away by someone else volunteering to take on the responsibilities of shepherding the EIP's repository."

Another way to look at it is that a Title is what you would use in a sentence when referring to the EIP.  "We should implement the `EVM Object Format` in Shanghai."  The description is what you would use to jog someone's memory as to what the EIP is.  "What is the `EVM Object Format`?" "It is the EIP that `Establishes contract versioning by way of a leading bytecode sequence.`" "Ah, right, I remember that one."

Reply:
I think actually your description is much better suited as opposed to the examples you gave :)

```
title: <Title is a few words, not a complete sentence.>
description: <Description is one full (short) sentence.>
abstract: <Abstract is multi-sentence (short paragraph) technical summary.>
```

Reply:
Works for me.  :)